app/assets/javascripts/components/mentee_application_step_3.js.coffee

```diff
@@ -21,15 +37,22 @@
   render: ->
     React.DOM.div
       className: 'fields'
-      React.createElement(ApplicationRadioGroup, key: 'programmingLanguage', field: 'programming_language', options: @props.programming_languages, placeholder: "Which programming language or framework would you like to learn during the mentorship program?", application: @props.application, setApplicationField: @props.setApplicationField)
+      React.createElement(ApplicationRadioGroup, key: 'programmingLanguage', field: 'programming_language', groupId: 'other', options: @props.programming_languages, placeholder: "Which programming language or framework would you like to learn during the mentorship program?", application: @props.application, setApplicationField: @props.setApplicationField)
+      React.createElement(ApplicationCheckboxGroup, key: 'otherProgrammingLanguages', field: 'other_programming_languages', groupId: 'opl', options: @props.programming_languages, placeholder: "To increase your chances of being matched to a suitable mentor, please consider specifying additional choices. (upto 4 choices)", application: @props.application, setApplicationField: @props.setApplicationField)
```

Comment:
I am only able to select one option. Here we can select 4 choices Max. Can we add that validation?

Reply:
The multi-selection works internally because when I check the database entry, multiple entries were stored. Seems like a styling issue. I will fix that and also add a validation for max 4 languages.

Reply:
Fixed stying issue and added the validation